README.md

```diff
@@ -1 +1,45 @@
-# Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Mobile version portfolio
+
+> This is a project for mobile version portfolio.
+
+
+## Built With
+
+- HTML 5
+- CSS 3
+
+
+## Getting Started
+
+**Just clone the project by runing the `https://github.com/bruk19/My-Portfolio.git` command in your command line, and you are done.**
+
+
+## Author
+
+👤 **Bruk Teshome**
+
+- GitHub: [@githubhandle](https://github.com/bruk19)
+- Twitter: [@twitterhandle](https://twitter.com/Bruktesh)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/bruk-teshome-ab4325226)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] license link is broken, kindly add .MIT license file to fix the link. Lets keep readme professional 💪🏻